client-src/elements/utils.js

```diff
@@ -400,3 +400,26 @@ export function checkMilestoneStartEnd(startEndPair, getFieldValue) {
     }
   }
 }
+
+
+export function checkFeatureNameAndType(getFieldValue) {
+  const name = (getFieldValue('name') || '').toLowerCase();
+  const featureType = Number(getFieldValue('feature_type') || '0');
+  const deprecationName =
```

Comment:
Since these are booleans, the variable names would be more clear as `isDeprecationName` and `isDeprecationType`.